ludwig/features/text_feature.py

```diff
@@ -247,12 +236,12 @@ def populate_defaults(input_feature):
 
 
 class TextOutputFeature(TextBaseFeature, SequenceOutputFeature):
-    def __init__(self, feature):
-        super().__init__(feature)
-        self.type = TEXT
+    decoder = 'generator'
 
+    def __init__(self, feature, decoder_obj=None):
+        TextBaseFeature.__init__(self, feature)
+        SequenceOutputFeature.__init__(self, feature)
         self.level = 'word'
```

Comment:
`level` `max_sequence_length` and `level` should all be class variables like decoder

Reply:
Missed those!